.yarnrc

```diff
@@ -1 +1,2 @@
 registry "https://registry.npmjs.com/"
+network-timeout 100000
```

Comment:
EOF

Reply:
Why do we use 100s?

Reply:
Set it to 60s to align npm's default timeout limitation.
By default:
yarn : 30s
npm: 60s

Now, we config them to the same timeout